src/Settings.h

```diff
@@ -10,6 +10,9 @@ class Settings
 
 public:
 
+    static bool userConfiguration;
+    static QString userConfigurationDir;
```

Comment:
If the setter is added, this would be of course in the private part. By the way, `...Dir` here seems like a leftover of a previous idea, it should be `...File`